draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -4555,7 +4563,7 @@ The below function outputs an appropriate Z for the Simplified SWU map ({{simple
 # - F, a field object, e.g., F = GF(2^521 - 1)
 # - A and B, the coefficients of the curve y^2 = x^3 + A * x + B
 def find_z_sswu(F, A, B):
-    R.<xx> = F[]                       # Polynomial ring over F
+    R.&lt;xx&gt; = F[]                       # Polynomial ring over F
```

Comment:
this change does not render as expected, since it's inside of a code block

Reply:
@armfazh Ack - I updated it because https://author-tools.ietf.org/ produces errors when it's 'xx'.  I